### PR TITLE
#27 Added changing roles and removing members from project.

### PR DESCRIPTION
**Pull request by gbalcewicz**

### CODE REVIEW

File: assets/js/pages/project/details/members.vue [hunk 1]
```diff
@@ -21,7 +22,45 @@
...
+                    <v-menu>
+                        <template v-slot:activator="{ on }">
+                            <v-btn icon v-on="on">
+                                <v-icon>more_vert</v-icon>
+                            </v-btn>
+                        </template>
+                        <v-list>
+                            <v-list-item link @click.prevent>
+                                <v-list-item-icon>
+                                    <v-icon>mdi-face</v-icon>
```
Comment (mariuszniemiec) [inline assets/js/pages/project/details/members.vue]: I am not completly sure, but maybe this icon should be aligned with top right dropdown "profile link" icon.

File: assets/js/pages/project/details/members.vue [hunk 2]
```diff
@@ -34,10 +73,13 @@
             </v-row>
             <v-row class="align-baseline flex-wrap">
                 <v-col class="flex-md-grow-1" cols="12" md="auto">
-                    <user-search :current-users="currentUsers" :selected.sync="newUsers" :disabled="updating"></user-search>
+                    <user-search :current-users="currentUsers" :selected.sync="newUsers"
+                                 :disabled="updating"></user-search>
                 </v-col>
                 <v-col class="d-flex justify-end flex-md-grow-0" cols="12" md="auto">
-                    <v-btn color="primary" @click="addUsers" :disabled="!newUsers.length || updating">Add selected users</v-btn>
+                    <v-btn color="primary" @click="addUsers" :disabled="!newUsers.length || updating">Add selected
```
Comment (mariuszniemiec) [inline assets/js/pages/project/details/members.vue]: Is it proper formatting? Maybe put down whole text in new line? Unless there is something I'm missing in community standards :)

File: assets/js/pages/project/details/members.vue (hunk 1 above)
Comment (gbalcewicz) [reply assets/js/pages/project/details/members.vue]: I had to use that face somewhere :D. Yes and no, in the top right icon shows editing profile. We can change this later.

File: assets/js/pages/project/details/members.vue (hunk 2 above)
Comment (gbalcewicz) [reply assets/js/pages/project/details/members.vue]: Autoformatting by phpstorm probably just broke the line at the limit. I'll fix this and move the member menu to separate component. Thanks for catching that!

**Pull request merged**